release-notes/README.md

```diff
@@ -1,5 +1,7 @@
 # Release Notes
 
+[1.0.3 released 12/13/1016](1.0/1.0.3.md)
```

Comment:
Could the time machine be turned on once more to retroactively fix the merge to master? /cc @leecow @blackdwarf 

> https://github.com/dotnet/core/blob/master/release-notes/README.md#release-notes
> 1.0.3 released 12/13/**1016**